native/CMakeLists.txt

```diff
@@ -113,11 +114,6 @@ if(NOT ANDROID AND NOT IOS AND NOT OHOS AND NOT OPENHARMONY)
     set(USE_WEBVIEW OFF)
 endif()
 
-if(OPENHARMONY)
-    set(USE_SE_V8 OFF)
-    set(USE_SE_NAPI ON)
-endif()
```

Comment:
NO NEEDED NOW. editor will pass the cmake flags to us:

Refer to https://github.com/cocos/creator-runtime-extensions/pull/470/files
```
    params.cMakeConfig.USE_SE_V8 = `set(USE_SE_V8 ${openHarmonyOS.useV8 ? 'ON' : 'OFF'})`;
    params.cMakeConfig.USE_SE_NAPI = `set(USE_SE_NAPI ${!openHarmonyOS.useV8 ? 'ON' : 'OFF'})`;
```